### DISCUSSION TITLE
Only offline-mode desired - worth it to use this package?

### DISCUSSION THREAD

**Discussion by ericdrosas87** [shadowwalker/next-pwa#466] [Q&A]
I am supporting a web client built with React/Next.js and we would like for the web app to be available offline. We don't necessarily want or need much of the other functionality that comes with a PWA. That being the case, does it make sense to implement this NPM package to make use of the offline functionality?

Alternatively, I am considering creating a proof of concept that basically just does a `postMessage()` to a service worker within the `getStaticProps()` method and in the service worker first check the Cache API for page data and load state from that, then if the API project can be reached compare the page data that is returned to what is loaded in state and update accordingly.

Has anyone found themselves in a similar situation with a similar use case? If so, how did you satisfy your use case - with this package or a custom implementation?

**Comment by ImBIOS** [comment]
@ericdrosas87 It seems this repo is dead and the maintainer cannot be contacted for so long now. I've made an immediate response by resurrecting this repo [ImBIOS/next-pwa](https://github.com/ImBIOS/next-pwa).
